### PR TITLE
DTSPO 382: Update tags cvp audio ingress

### PR DESCRIPTION
**Pull request by madjava**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPO-382

### Change description ###
Update tagging for cvp respources

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: pipeline/steps/tf-plan.yaml [hunk 1]
```diff
@@ -137,7 +139,31 @@ steps:
...
+        -var "location=$(location)"
+        -var "product=$(product)"
+        -var "builtFrom=${{ parameters.builtFrom }}"
+        -var "environment=${{ parameters.env }}"
+        -var "env=${{ parameters.env }}"
+        -var "cert_path=$(certPath)"
+        -var "service_certificate_kv_url=$(secretId)"
+        -var "thumbprint=$(thumbprint)"
+        -var "key_vault_id=$(kvId)"
+        -var "wowza_sku=$(wowza_sku)"
```
Comment (timja) [inline pipeline/steps/tf-plan.yaml:152]: this is pretty crazy, thanks for re-formatting at least it's much more readable if still crazy having all these vars set here.

**Comment by timja** [comment]
For some reason this is commented out https://github.com/hmcts/cvp-audio-ingress/blob/master/modules/wowza/az-monitor-diagnostic-set.tf#L48-L70

causing it to try delete which doesn't work because the scope is locked (removed in https://github.com/hmcts/cvp-audio-ingress/pull/52)

@lowflying you able to take a look at this please?

**Review by timja — COMMENTED** [review]
all PR changes look good, but build needs to be green and plan closely checked

**Review by lowflying — APPROVED** [review]
They are some BEAUTIFUL tags in a repo of nettles.
LGTM

File: pipeline/steps/tf-plan.yaml [hunk 2]
```diff
@@ -82,7 +82,7 @@ steps:
         az keyvault secret download --id https://$(infra_kv).vault.azure.net/secrets/$(dynatrace_token) --file dynatrace-token
 
         echo "Import Dynatrace env token to KV $kvName"
-        az keyvault secret set --name $(dynatrace_token) --file dynatrace-token --vault-name $kvName --verbose
+        az keyvault secret set --name $(dynatrace_token) --file dynatrace-token --vault-name $kvName --tags $tags --verbose
```
Comment (timja) [inline pipeline/steps/tf-plan.yaml:85]: no need to set tags on secrets, they are used for different things

**Review by timja — APPROVED** [review]

**Pull request merged**